api/pyproject.toml

```diff
@@ -35,3 +35,194 @@ skip = ['migrations','.venv','.direnv']
 
 [tool.pytest.ini_options]
 addopts = ['--ds=app.settings.test', '-vvvv', '-p', 'no:warnings', '-n', 'auto']
+
+[tool.poetry]
+name = "y"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.11"
+django = "3.2.19"
+analytics-python = "1.4.0"
+appdirs = "1.4.4"
+asgiref = "3.5.0"
+azure-core = "1.23.1"
+azure-identity = "1.12.0"
+boto3 = "1.26.75"
+botocore = "1.29.75"
+cachetools = "4.1.1"
+certifi = "2022.12.7"
+cffi = "1.15.1"
+chargebee = "2.7.7"
```

Comment:
I'd use more loose version constraints, e.g. the [tilde requirements](https://python-poetry.org/docs/dependency-specification/#tilde-requirements) or even the caret requirements. I certainly don't see value in pinning the patch versions. @matthewelwell thoughts?

Reply:
I was earlier planning to specify dependencies in the `.in` file one by one, starting with django<4, but it seems the projects work only with 3.2.x django versions. Manually adding main dependencies in `.in` using `poetry add` led to some conflicts towards the end, So I went ahead and added all the dependencies from the `.txt` files with their exact version.

Reply:
I have updated the dependencies to include only main dependencies as mentioned in `.in` files